main.tf

```diff
@@ -0,0 +1,74 @@
+# Email Identities
+resource "aws_ses_email_identity" "this" {
+  for_each = var.email_addresses
+  email    = each.key
+}
+
+# Domain Identity with Route53 validatation
+resource "aws_ses_domain_identity" "this" {
+  domain = var.domain_name
+}
+
+resource "aws_ses_domain_dkim" "this" {
+  domain = aws_ses_domain_identity.this.domain
+}
+
+data "aws_route53_zone" "this" {
+  name         = aws_ses_domain_identity.this.domain
+  private_zone = false
+}
+
+resource "aws_route53_record" "dkim" {
```

Comment:
can we remove route53 out of equation? there is a huge portion of customers who won't transfer their domain to AWS because of "reasons", module should provide you with the outputs that end-user might take and import to their DNS zone - route53 can be example though

Reply:
single dns record output should have 3(+1 if we should provide recomended TTL) components:
- name 
- type (i.e. CNAME, TXT, ...)
- value 
- TTL (optional but it's good to have one)

Reply:
Route 53 has been moved from the main module to the example and the outputs have been reorganized. I didn't add TTL parameter because it's not used in the SES module and doesn't have any impact on the verification attributes.